src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -33,6 +33,30 @@
         <property name="username" value="ca"/>
         <property name="password" value=""/>
     </bean>
+
+
+    <bean id="flightDAO" class="com.epam.lowcost.DAO.FlightDAOImpl">
+        <constructor-arg ref="idDataSource"/>
```

Comment:
Use bean with id "dataSource" instead of "idDataSource" (see other DAO classes in dev branch)

Reply:
done